es6-destructuring/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML Skeleton</title>
+</head>
+
+<body>
+
+  <script>
+    const book1 = {
+  title: 'Goodnight Punpun',
+  author: 'Inio Asano',
+  libraryID: 3353
+};
+    const { title, author, libraryID } = book1;
+    console.log('The title of the book is ' + title + ', ' + 'the author is ' + author + ', ' + 'and the library id is ' + libraryID + '.');
+
+const book2 = {
+  title: 'Les Fleurs du mal',
+  author: 'Charles Baudelaire',
+  libraryID: 2345
+};
+    const { otherTitle = book2.title, otherAuthor = book2.author, otherLibraryID = book2.libraryID } = book2;
+    console.log(`The title of the book is ${otherTitle}, the author is ${otherAuthor}, and the library id is ${otherLibraryID}.`);
+
+    const library = [
+      {
+        title: 'The Road Ahead',
+        author: 'Bill Gates',
+        libraryID: 1254
+      },
+      {
+        title: 'Walter Isaacson',
+        author: 'Steve Jobs',
+        libraryID: 4264
+      },
+      {
+        title: 'Mockingjay: The Final Book of The Hunger Games',
+        author: 'Suzanne Collins',
+        libraryID: 3245
+      },
+      {
+        title: "Dead Dead Demon's De De De De Destruction",
+        author: 'Inio Asano',
+        libraryID: 1233
+      }
+    ];
+
+    const book3 = library[0];
```

Comment:
Lines 50, 51, 52 are not using the destructuring technique. Please review after Q&A :)